bokeh/embed/notebook.py

```diff
@@ -44,8 +47,10 @@
 #-----------------------------------------------------------------------------
 # Dev API
 #-----------------------------------------------------------------------------
+Theme = Union[Theme, FromCurdoc, None]
```

Comment:
I think `ThemeSource` might be a better name here, in any case I don't think it can be `Theme` since that shadows the `Theme` import above.

Reply:
Including `None` is unnecessary, because the default is always (as far as I can tell) `FromCurdoc`. Even if `None` was used, it would be usually preferred to use `Optional[ThemeSource]`. Another issue is passed around as a class `FromCurdoc` and the above type indicates an instance of `FromCurdoc`. The correct type is:
```
Union[Theme, Type[FromCurdoc]]
```
(`Type` comes from `typing` module). There's an orthogonal question whether this is the right way of working with singleton objects altogether, but that's something we will need to answer later on.